hpobench/benchmarks/nas/nasbench_101.py

```diff
@@ -109,17 +132,52 @@ def objective_function(self, configuration: Union[CS.Configuration, Dict],
             info : Dict
                 fidelity : used fidelities in this evaluation
         """
+
+        if isinstance(run_index, int):
+            assert 0 <= run_index <= 2, f'run_index must be in [0, 2], not {run_index}'
+            run_index = (run_index, )
+        elif isinstance(run_index, tuple) or isinstance(run_index, list):
+            assert len(run_index) != 0, 'run_index must not be empty'
+            assert min(run_index) >= 0 and max(run_index) <= 2, \
+                f'all run_index values must be in [0, 2], but were {run_index}'
+        elif run_index is None:
+            logger.debug('The run index is explicitly set to None! A random seed will be selected.')
+            run_index = tuple(np.random.choice((0, 1, 2), size=1))
```

Comment:
You should use self.rng for this

Reply:
And l151 should have happened before this

Reply:
I've also changed this for the other nas benchmarks. (rng as first operation + None selects a seed randomly)